meta-luneos/recipes-support/sensorfw/sensorfw/0001-Add-trustlevel-for-enhanced-ACG.patch

```diff
@@ -3,6 +3,8 @@ From: Herman van Hazendonk <github.com@herrie.org>
 Date: Tue, 13 Dec 2022 15:16:50 +0100
 Subject: [PATCH] com.nokia.SensorService.role.json: Add trustLevel
 
+Upstream-Status: Pending [see https://github.com/sailfishos/sensorfw/pull/17]
```

Comment:
that's Submitted

read https://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines#Patch_Header_Recommendations:_Upstream-Status

Reply:
And please remove incorrect Upstream-Statuses from .bb file.